cgmes/cgmes-conformity/src/test/java/com/powsybl/cgmes/conformity/test/CgmesConformity1NetworkCatalog.java

```diff
@@ -966,8 +966,8 @@ private static void addPhaseTapChanger(
             PhaseTapChanger.RegulationMode mode, boolean regulating,
             double regulationValue, double targetDeadband) {
         LOG.debug("EXPECTED tx {}", tx.getId());
-        double rho0 = tx.getRatedU2() / tx.getRatedU1();
-        double rho02 = rho0 * rho0;
+        double ratio0 = tx.getRatedU2() / tx.getRatedU1();
```

Comment:
The ratio in CGMES is primary nominal voltage / seconday nominal voltage, no ?

Reply:
It is the structural ratio at end2 (after transmission impedance) so in this case the ratio is secondary nominal voltage / primary nominal voltage.
The structural ratio at end1 (before transmission impedance) is primary nominal voltage / secondary nominal voltage as you said but in the current CGMES conversion the structural ratio is always considered at end2 (after transmission impedance)